docs: Add new glossary items: ML, API, Vision Service, Machine ID, Pi, flashed (#4326)
viamrobotics/docs: docs/dev/reference/glossary/pi.md

---
title: Pi
id: pi

> full_link: https://www.raspberrypi.com/

Reviewer: usually this is to a viam docs page.... this feels like we're advertising pis too much

> full_link: https://www.raspberrypi.com/

Reviewer 2: ```suggestion
full_link:
```

short_description: Short for Raspberry Pi, a series of small single-board computers developed by the Raspberry Pi Foundation.
aka:
type: "page"
---

> **Pi** is short for **Raspberry Pi**, a series of small, affordable single-board computers developed by the Raspberry Pi Foundation. These credit card-sized computers are widely used in robotics, IoT projects, education, and DIY electronics.

Reviewer: ```suggestion
Pi is short for Raspberry Pi, a series of small, affordable single-board computers developed by the Raspberry Pi Foundation.
These credit card-sized computers are widely used in robotics, IoT projects, education, and DIY electronics.
```
nit incl. newline

> The Viam platform supports Raspberry Pi devices as host computers for running `viam-server`, making them popular choices for building robots and smart machines.

Reviewer: ```suggestion
Viam supports many different devices including Raspberry Pis as host computers for running `viam-server`.
```

> **Pi** is short for **Raspberry Pi**, a series of small, affordable single-board computers developed by the Raspberry Pi Foundation. These credit card-sized computers are widely used in robotics, IoT projects, education, and DIY electronics.
> 
> The Viam platform supports Raspberry Pi devices as host computers for running `viam-server`, making them popular choices for building robots and smart machines.

Reviewer: Or shorten this way up:

```suggestion
Pi is short for Raspberry Pi, a series of single-board computers developed by the Raspberry Pi Foundation.
`viam-server` can run on a [variety of computers](/operate/get-started/basics/#what-platforms-does-viam-run-on); Raspberry Pi is just one option.
```

> When someone mentions "flashing a Pi" or "setting up a Pi," they're referring to installing an operating system or software on a Raspberry Pi device.

Reviewer: ```suggestion
```
feels like overkill
